calico/openstack/test/test_plugin_etcd.py

```diff
@@ -129,7 +152,15 @@ def etcd_read(self, key, recursive=False, timeout=None):
             print "children: %s" % [child.key
                                     for child in read_result.children]
             if read_result.value is None and read_result.children == []:
-                raise lib.m_etcd.EtcdKeyNotFound()
+                raise lib.m_etcd.EtcdKeyNotFound(self.etcd_data)
+
+            read_result._children = []##actual direct children of the dir in etcd response. needed for status_dir, where children are dirs and gotta be iterated
+            list_of_statuses = [{"key": K} for K in self.etcd_data.keys()]
+            read_result._children.append({"nodes": list_of_statuses})
+            #takto vyzeraju _children
+            # {'nodes': [list of status]}
+            # kde status:  {'key': '/../uptime'}
```

Comment:
I don't speak Slovak, so this comment isn't useful to me. Please could you rewrite it in English?